assets/js/keyBindings.coffee

```diff
@@ -87,7 +87,8 @@ if module?
     ENTER_VISUAL      : ['v']
     ENTER_VISUAL_LINE : ['V']
 
-    EXPORT            : ['ctrl+s']
+    EXPORT_FILE       : ['ctrl+s'] # Really bad shortcuts, change
```

Comment:
hmm, yeah, not sure.  I kinda feel like users are unlikely to remember shortcuts for import.  possibly some kind of UI makes more sense?  maybe paul has thoughts